scripts/builtin/vectorToCsv.dml

```diff
@@ -32,10 +32,10 @@
 # indexes  indexes
 # ----------------------------------------------------------------------------------------
 
-m_vectorToCsv = function(Matrix[Double] mask)
+m_vectorToCsv = function(Matrix[Double] mask = as.matrix(0), Integer n=1)
```

Comment:
the error is here, or rather the parsers we have in Python does not like this syntax.